Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionPropertiesView.cs

```diff
@@ -41,6 +42,12 @@ public override void RedrawUI((SerializedInputAction ? , List<string>) viewState
             {
                 tooltip = inputAction.actionTypeTooltip
             };
+
+            // Tighten up the gap between the label and dropdown so the latter is more readable when the parent pane is at min width.
+            var actionLabel = actionType.Q<Label>();
+            actionLabel.style.minWidth = m_DropdownLabelWidth;
```

Comment:
Can this be in the uxml rather than the code?

Reply:
I wondered about that, but there doesn't seem to be uxml for the Properties view, it's entirely programmatic - Not sure if there's a nice way around that?

Reply:
Fair enough, let's move forward with this then.